install

```diff
@@ -94,8 +94,12 @@ end
 def sudo?
   return @have_sudo unless @have_sudo.nil?
 
-  Kernel.system "/usr/bin/sudo", "-v"
-  @have_sudo = $CHILD_STATUS && $CHILD_STATUS.success?
+  begin
+    Kernel.system "/usr/bin/sudo", "-l", "mkdir"
+    @have_sudo = $CHILD_STATUS.success?
+  rescue Interrupt
+    exit!
+  end
```

Comment:
Is the `begin/rescue/end` block needed? Please remove it if not.

Reply:
If this does not exist,
Ctrl-C shows traceback message.

Reply:
Thanks for the explanation. I believe `exit` should work and also prevent the traceback. Let's use that rather than `exit!`.

Reply:
I think I misunderstood your comment.
`begin`~`end` is not necessary here, so I removed it.